modules/quanthub_core/quanthub_core.routing.yml

```diff
@@ -5,3 +5,24 @@ powerbi_embed_config:
     _controller: '\Drupal\quanthub_core\Controller\PowerBIEmbedController::postData'
   requirements:
     _permission: 'access content'
+
+quanthub_core.export_to_file:
+  path: '/powerbi/reports/{reportId}/exportTo'
+  defaults:
+    _controller: '\Drupal\quanthub_core\Controller\PowerBiApiController::proxyPowerBiRequest'
+  requirements:
```

Comment:
Should we add "_method: GET" to validate method on our side to avoid send wrong requests to PBI side?